scripts/lua/modules/graph_utils.lua

```diff
@@ -932,20 +932,21 @@ end
 
 -- ########################################################
 
-function createActivityRRDCounter(path, step, verbose)
+function createActivityRRDCounter(path, verbose)
    if(not(ntop.exists(path))) then
       if(verbose) then io.write('Creating RRD '..path..'\n') end
       local prefs = ntop.getPrefs()
-      local hb = step * 2 -- Default hb = 2 minutes
+      local step = 300
+      local hb = step * 2
       ntop.rrd_create(
 	 path,
 	 step,
 	 'DS:in:DERIVE:'..hb..':U:U',
 	 'DS:out:DERIVE:'..hb..':U:U',
 	 'DS:bg:DERIVE:'..hb..':U:U',
-         'RRA:AVERAGE:0.5:1:'..tostring(prefs.other_rrd_raw_days*24*(3600/step)),
```

Comment:
I've changed pref from days to hours to possibly reduce RRA sizes. Now it only takes 196K per host (versus old 3 MB).